### PR TITLE
Remove all dependencies, update package CI.

### PR DESCRIPTION
**Pull request by rhpvorderman**
Also added PEP-518 style packaging with pyproject.toml and fixed a few tests that were outdated (new alleles have been discovered). 

I also added tests for the package and the documentation.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -15,18 +15,26 @@
...
 
-project = 'aav'
-copyright = '2018, Author'
-author = 'Author'
+# Get package information from the installed package.
+package = pkg_resources.get_distribution("array_as_vcf")
+metadata_file = Path(package.egg_info) / Path(package.PKG_INFO)
+metadata = DistributionMetadata(path=str(metadata_file))
+
+project = 'python-isal'
```
Comment (Redmar-van-den-Berg) [inline docs/conf.py:30]: Copy/paste error?

**Review by Redmar-van-den-Berg — CHANGES_REQUESTED** [review]
Looks good, my only comment is that it looks like there is a copy/paste error in docs/conf.py, since this project has nothing to do with 'python-isal'

File: docs/conf.py (hunk 1 above)
Comment (rhpvorderman) [reply docs/conf.py:30]: Eh. Yes. Whoops.

Comment (rhpvorderman) [reply docs/conf.py]: Fixed, thanks for the review!

**Review by Redmar-van-den-Berg — APPROVED** [review]

**Pull request merged**